### PR TITLE
views: fix actions blueprint for multiple entries

### PR DESCRIPTION
**Pull request by jma**
* remove useless code
* indentation corrections
* bad indentation for return value

Signed-off-by: Johnny Mariéthoz <Johnny.Mariethoz@rero.ch>

### CODE REVIEW

**Comment by zzacharo** [comment]
@jma https://github.com/inveniosoftware/invenio-circulation/pull/47/files#diff-2b66d1d23d98cd75cd5ae11e29cb6183L117 good catch thanks!

**Review by zzacharo — APPROVED** [review]

**Comment by zzacharo** [comment]
@jma I am thinking that the probably we should put the LoanActions view only in loan_pid rest. If you extend the endpoints probably it doesn't make sense to register actions there. wdyt?

**Comment by jma** [comment]
@zzacharo yes you are right. In such case we need to select the right option. I will try to solve it.

**Comment by jma** [comment]
@zzacharo I think it is solved.

**Comment by zzacharo** [comment]
@jma I think that we should just hardcode it inside the build method. It's a bit confusing and actually we don't gain something from having it as `_CIRCULATION_LOAN_PID_TYPE`. WDYT?

**Comment by jma** [comment]
@zzacharo why using it `api.py` and not in `views.py`?

**Comment by zzacharo** [comment]
@jma but it's more views related not api.

**Comment by zzacharo** [comment]
@jma also can you rebase? When you do it let me know and we can merge it.

**Comment by zzacharo** [comment]
@jma I messed up your pr with another merge commit. I tried to update the branch from here. Sorry for that! Can you just rebase and push force in your pr again?

File: invenio_circulation/views.py [hunk 1]
```diff
@@ -126,7 +127,6 @@ def __init__(self, serializers, ctx, *args, **kwargs):
         """Constructor."""
         super(LoanActionResource, self).__init__(
             serializers,
-            default_media_type=ctx.get('default_media_type'),
```
Comment (jma) [inline invenio_circulation/views.py:129]: `default_media_type` is not in `ctx`, thus this line is probably useless.

Comment (zzacharo) [reply invenio_circulation/views.py:129]: Yes we could omit this line for now.

**Comment by zzacharo** [comment]
My only comment is if you want to hardcode the value `loan_pid` in the `build_blueprint_with_loan_actions ` instead of the config. Other than that I am ok with this PR to be merged.

**Pull request merged**